adding_species_manuscript.md

```diff
@@ -54,51 +61,63 @@ into the stdpopsim catalog in order to make it available
+Models inferred from these tools are an important community resource, but reuse of such models can be arduous and error-prone (@Ragsdale2020).
 
 Simulations from population genomic models have many uses, both for methods development and empirical research.
-They provide training data for inference methods based on machine learning @Schrider2018 or Approximate Bayesian Computation @Csillery2010.
-They can also serve as baselines for further analyses; for example, models incorporating demographic history serve as null models in selection analyses @Hsieh2016a.
-(RNG: This self-cite might not be the best reference.)
-Population genomic simulations can also help guide conservation decisions for threatened species @Teixeira2021.
+They provide training data for inference methods based on machine learning (@Schrider2018) or Approximate Bayesian Computation (@Csillery2010).
+They can also serve as baselines for further analyses; for example, models incorporating demographic history serve as null models in selection analyses (@Hsieh2016a).
+Population genomic simulations can also help guide conservation decisions for threatened species (@Teixeira2021).
 
-In general, population genomic simulations become more useful as they incorporate more features of the organism's biology.
-The demographic history of a species, encompassing population sizes, divergences, and gene flow, can dramatically affect patterns of genetic variation @Teshima2006.
-(RNG: Maybe not the best citation for this.)
-Recombination rate variation across the genome also strongly affects genetic variation and haplotype structure @Nachman2002, particularly when linked selection is important.
+In general, population genomic simulations become more useful the more realistically they represent the organism being simulated - that is, as they incorporate more features of the organism's biology.
+The demographic history of a species, encompassing population sizes, divergences, and gene flow, can dramatically affect patterns of genetic variation (@Teshima2006), and recombination rate variation across the genome also strongly affects genetic variation and haplotype structure (@Nachman2002), particularly when linked selection is important. 
+Thus estimates of these parameters are fundamentally important to the development of simulations of species of interest. This presents challenges not only in the coding of the simulations themselves, but in the choice of parameter estimates to use to shape the simulation model.
 
-Stdpopsim is a community resource recently developed to provide easy access to detailed population genomic simulations @Adrion2020.
+Stdpopsim is a community resource recently developed to provide easy access to detailed population genomic simulations (@Adrion2020).
 This resource lowers the technical barriers to performing such simulations and reduces the possibility of erroneous implementation of established models.
```

Comment:
do established models mean that they have been published? I would say published models instead.